lib/circuitry/processors/forker.rb

```diff
@@ -7,9 +7,11 @@ class << self
         include Processor
 
         def process(&block)
+          on_exit = Circuitry.subscriber_config.on_fork_exit
```

Comment:
We can wrap this `on_exit` logic in a helper method within the `Processor` module since `on_fork_exit` and `on_batch_exit` can be unified.